recipes/r-rcpp/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.12.10' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rcpp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: Rcpp_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/Rcpp_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/Rcpp/Rcpp_{{ version }}.tar.gz
+
+
+  sha256: 4ef811a87be7d4d6c2d9084c5a23a3267192bcd79b949ba8ad8ff7a70a221781
+  patches:
+    - 0001-Add-CXX1XSTD-to-PKG_CPPFLAGS.patch
+    - 0002-macOS-When-target-lt-10.9-libstdc-supports-only-some.patch
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('Rcpp')"  # [not win]
+    - "\"%R%\" -e \"library('Rcpp')\""  # [win]
+
+about:
+  home: http://www.rcpp.org, http://dirk.eddelbuettel.com/code/rcpp.html, https://github.com/RcppCore/Rcpp
+  license: GPL (>= 2)
```

Comment:
Is there a license file we can include?

ref: https://conda.io/docs/building/meta-yaml.html#license-file